gradleutils/test_config.gradle

```diff
@@ -0,0 +1,18 @@
+dependencies {
```

Comment:
This gradle utility file share the common dependency across all the UI instrumentation tests